beacon/validator/src/test/java/tech/pegasys/teku/validator/coordinator/BlockOperationSelectorFactoryTest.java

```diff
@@ -597,6 +677,17 @@
                     Optional.empty())));
   }
 
+  private void prepareCachedBuilderPayload(
+      final UInt64 slot,
+      final ExecutionPayload executionPayload,
+      final tech.pegasys.teku.spec.datastructures.builder.BlobsBundle blobsBundle) {
```

Comment:
maybe make it Optional or clarify method name that it's postDeneb only?

Reply:
made it optional